frontend/src/features/settings/account/EmailAddressForm.tsx

```diff
@@ -134,33 +134,30 @@ export const EmailAddressForm = () => {
             </Box>
           )}
           <form onSubmit={handleSubmit}>
-            <Grid container spacing={2} direction="column" alignItems="stretch">
-              <Grid item md={6}>
-                <FormTextField
-                  required
-                  fullWidth
-                  label="New email address"
-                  name="email"
-                  type="email"
-                  formError={formError}
-                />
-              </Grid>
-              <Grid item md={6}>
-                <Button
-                  type="submit"
-                  color="secondary"
-                  fullWidth
-                  variant="contained"
-                >
-                  Send verification email
-                </Button>
-              </Grid>
-            </Grid>
+            <FormTextField
+              required
+              fullWidth
+              label="New email address"
+              name="email"
+              type="email"
+              formError={formError}
+              style={{ marginBottom: 16 }}
+            />
+            <Button
+              type="submit"
+              color="secondary"
+              fullWidth
+              variant="contained"
+            >
+              Send verification email
+            </Button>
           </form>
-        </div>
-      </>
+        </Grid>
+      </Grid>
     );
   };
 
   return <Box minHeight="180px">{getContent()}</Box>;
 };
+
+export default EmailAddressForm;
```

Comment:
Re-structured a little bit this component so that it is more similar to other forms on the same page. Previously it was the only component that may take more than `md={6}` on large screens

Reply:
Nothing critical, but it was actually intentional to allow the information message to fill the remaining width.

With `md={6}` on the whole section, the message  block may be unnecessarily narrow on some screens:
![image](https://user-images.githubusercontent.com/4726554/142005959-f848abe8-ecff-438c-a508-721e22711d94.png)

Reply:
We can see from your screenshot that it is not the only problem that we have for the layout on md screens.

I propose to go ahead with the current change proposed in this PR, then clean-up the whole layout. I created a ticket for it #306